backend/src/processing/mod.rs

```diff
@@ -177,7 +177,10 @@ impl EventProcessor {
     /// Handle different event types
     async fn handle_event_types(&self, processed: &mut ProcessedEvent) -> Result<()> {
         let event_name = processed.event.raw.event_name.clone();
-        if processed.event.raw.is_custom_event {
+        if event_name.eq_ignore_ascii_case("cwv") {
+            processed.event_type = "cwv".to_string();
+            processed.custom_event_json = processed.event.raw.properties.clone();
+        } else if processed.event.raw.is_custom_event {
             processed.event_type = "custom".to_string();
             processed.custom_event_name = event_name;
             processed.custom_event_json = processed.event.raw.properties.clone();
```

Comment:
This is a bit stupid design choice from my side. But this event name check should be checked after checking if the event is a custom event to prevent clashing in case of users using a custom event name called `cwv` 😄 It's definitely something I want to update soon, such that this is not necessary. I've just opened an issue for this https://github.com/betterlytics/betterlytics/issues/473